cli/src/deploy/mod.rs

```diff
@@ -112,19 +110,19 @@ async fn get_credentials(token: &str) -> HashMap<String, CredentialsConfig> {
   let path = Path::new(file_name);
   let file = File::open(path);
   if let Err(err) = file {
-    eprintln!("Cannot access credentials at {}. Error: {}", file_name, err);
+    let err_str = format!("Cannot access credentials at {}. Error: {}", file_name, err);
+    eprintln!("{}", err_str);
```

Comment:
Is this necessary? Wouldn't `eprintln!(err_str);` work?

Reply:
No, must be a string literal.